owner/manager.go

```diff
@@ -259,6 +259,15 @@ func (m *ownerManager) watchOwner(ctx goctx.Context, etcdSession *concurrency.Se
 func init() {
 	err := setManagerSessionTTL()
 	if err != nil {
-		log.Warnf("[ddl] set manager session TTL failed %v", err)
+		log.Warnf("set manager session TTL failed %v", err)
+	}
+}
+
+func isContextDone(ctx goctx.Context) bool {
```

Comment:
Nope. In fact, this function also exists in DDL package, but it seems awkward to export it here. Any advice?